doc/website/advanced/configuration-guide.md

```diff
@@ -142,12 +169,19 @@ size = 1024
 count = 100
 ```
 
-When no config file is specified, a hard-coded version similar to the [default config](https://github.com/eclipse-iceoryx/iceoryx/blob/master/iceoryx_posh/etc/iceoryx/roudi_config_example.toml) will be used.
+When no configuration file is specified a hard-coded version similar to the 
+[default config](https://github.com/eclipse-iceoryx/iceoryx/blob/master/iceoryx_posh/etc/iceoryx/roudi_config_example.toml)
+will be used.
 
 ### Static configuration
 
-Another way is to have a static config that is compile-time dependent, this means that you have to recompile your RouDi application if you want to change your config (not the iceoryx_posh_roudi lib).
-You can have your source file with `main()` method where you can create your custom configuration and pass it to a RouDi instantiation.
-In your CMake file for your custom RouDi you need to ensure that it is **not** linking against `iceoryx_posh_config` to have a static config.
+Another way is to have a static configuration that is compiled into the roudi application.
+As a consequence you have to recompile your RouDi application if you want to change
+your config (not the `iceoryx_posh_roudi` lib).
+You can create your custom configuration in the `main()` function where you then pass
+it as constructor argument to the RouDi instance.
+In the cmake file entry of the custom RouDi executable you need to ensure that it
+is **not** linking against `iceoryx_posh_config` to ensure using the static configuration.
 
-A good example of a static config can be found [here](https://github.com/eclipse-iceoryx/iceoryx/blob/master/iceoryx_examples/iceperf/roudi_main_static_config.cpp).
+An example of a static config can be found 
+[here](https://github.com/eclipse-iceoryx/iceoryx/blob/master/iceoryx_examples/iceperf/roudi_main_static_config.cpp).
```

Comment:
```suggestion
[here](https://github.com/eclipse-iceoryx/iceoryx/blob/v2.0.0/iceoryx_examples/iceperf/roudi_main_static_config.cpp).
```

Reply:
Since it is merged to master I would leave it has it is.